### DISCUSSION TITLE
Support packer host behind proxy

### DISCUSSION THREAD

**Discussion by sestegra** [vmware/packer-examples-for-vsphere#39] [Ideas]
Hello,

Would you accept PR for proxy support ?
* For Linux, `ssh_proxy_host` and `ssh_proxy_port` are required.
* For Windows, environement variables are enough when launching packer command line.

On top of that, when using proxy, `http_content` doesn't work, so it need to be replaced by `cd_content` and boot option should be updated.
I don't know if all Linux distribution support CD instead of HTTP server for install configuration.

Kind regards,
Stéphane

**Comment by tenthirtyam** [comment]
I'm already updating all the Linux examples to remove the requirement for the HTTP server on the Packer host. I've already completed it for RHEL and it's variants. I plan to have that in the 20.10 update. Feel free to submit an enhancement to support a bastion/proxy.

Ryan

**Comment by tenthirtyam** [comment]
By the way, I can confirm that PhotonOS doesn't support kickstart from cdrom. 😢 Seems to be my only blocker at the moment.

Ryan

**Comment by tenthirtyam** [comment]
Tracking under PR https://github.com/rainpole/packer-vsphere/pull/40.

Ryan

**Comment by tenthirtyam** [comment]
Committed to `main`: https://github.com/rainpole/packer-vsphere/pull/40.

Ryan
